src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -156,6 +159,12 @@ public void OnPersist(StoreView snapshot, IReadOnlyList<Blockchain.ApplicationEx
                     HandleNotification(snapshot, transaction, notifyEventArgs.ScriptHash, stateItems,
                         nep5BalancesChanged, ref transferIndex);
                 }
+
+                // Handle SystemFee and NetworkFee GAS burn for each transaction
+                if (appExecuted.Transaction is null) continue;
+                HandleNotification(snapshot, appExecuted.Transaction, NativeContract.GAS.Hash,
+                    new VM.Types.Array(new VM.Types.StackItem[] { "Transfer", appExecuted.Transaction.Sender.ToArray(), VM.Types.StackItem.Null, appExecuted.Transaction.SystemFee + appExecuted.Transaction.NetworkFee }),
+                    nep5BalancesChanged, ref transferIndex);
```

Comment:
I think we should not handle the burn events for each transaction, but for the `SYSTEM` trigger only.

https://github.com/neo-project/neo/blob/c1b9b9712ee638768014150e2369ac76e02863db/src/neo/Ledger/Blockchain.cs#L504

Reply:
Here the ScriptContainer (transaction) of the ApplicationEngine with the SYSTEM trigger is null.
https://github.com/neo-project/neo/blob/c1b9b9712ee638768014150e2369ac76e02863db/src/neo/Ledger/Blockchain.cs#L499.
The reason we trace each transaction is to refresh the sender's balance and the transfer history.

Reply:
I wonder whether we should record the transfer history for non-tranfer transactions. If not, maybe we should just loose the restrictions on `notifyEventArgs.ScriptContainer is Transaction transaction`, just to trace the balance change.
https://github.com/neo-project/neo-modules/blob/c75c0766fc15832fe3cd5edce4c3bb4b68e56253/src/RpcNep5Tracker/RpcNep5Tracker.cs#L156